### PR TITLE
test(input): migrate to generators

### PR DESCRIPTION
**Pull request by liamdebeasi**
Issue number: N/A

---------

 

 

## What is the current behavior?

Input tests use legacy syntax

## What is the new behavior?

- Input tests use modern syntax

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by liamdebeasi** [comment]
Going to wait for at least 2 reviews before merging since this is another big change.

**Review by thetaPC — APPROVED** [review]
LGTM

File: core/src/components/input/test/item/input.e2e.ts
Comment (amandaejohnston) [inline core/src/components/input/test/item/input.e2e.ts:46]: Optional: Looks like this one could be pulled into a separate block and only run on LTR/one mode.

File: core/src/components/input/test/label-placement/input.e2e.ts
Comment (amandaejohnston) [inline core/src/components/input/test/label-placement/input.e2e.ts:4]: Optional: Could the identical `configs()` loops be merged into one here?

**Review by amandaejohnston — APPROVED** [review]

File: core/src/components/input/test/item/input.e2e.ts
Comment (liamdebeasi) [reply core/src/components/input/test/item/input.e2e.ts]: I ended up making this a spec test. Good catch!

**Pull request merged**
